Source/SvgElement.cs

```diff
@@ -844,10 +844,8 @@ protected virtual void Render(ISvgRenderer renderer)
         /// <param name="renderer">The <see cref="ISvgRenderer"/> to render the child <see cref="SvgElement"/>s to.</param>
         protected virtual void RenderChildren(ISvgRenderer renderer)
         {
-            foreach (SvgElement element in this.Children)
-            {
+            foreach (var element in Children)

```

Comment:
Any specific reason why to pick var instead of SvgElement? When writing new code I can imagine picking var, but any reason why you changed an existing explicit declaration to a var?

Reply:
@gvheertum 

It is basically replaced with `var`.

Reply:
@H1Gdev - this does not answer the question 😉 
I will merge now, but I agree with @gvheertum that we don't need to adapt all working code to our own coding style 😜

Reply:
I didn't understand your question.

This time I was reviewing the processing of `Render` method.
There was a change to `var` as part of this change.

Certainly, this method does not change anything from the original source code.
So I will revert to original source code.